pipeline_steps/common.groovy

```diff
@@ -1915,6 +1919,44 @@ void createRelease(String component_text, Boolean from_rc_branch){
   )
 }
 
+void updateDependents(REPO_NAME){
+  parallelBuilds = [:]
+  allWorkflowJobs = Hudson.instance.getAllItems(WorkflowJob)
+  venv = "${WORKSPACE}/.componentvenv"
+
+  // get components that are dependent on this repo
+  def dependents = sh(
+    returnStdout: true,
+    script: """#!/bin/bash
+      set +x; . ${venv}/bin/activate; set -x
+      component --releases-dir ./ dependents --component-name ${REPO_NAME} --text
+    """
+  ).tokenize()
+
+  dependents.add($REPO_NAME) //add this release to be dep updated
```

Comment:
True, True...   but REPO_NAME is being released, right?  it was decided that we want to do a dep update on the released project as well, hence why it is being added to the list to be dep updated.  Or please further explain in slack if I'm making some improper assumptions.  thanks

Reply:
The component is having a new release created but that is not the same thing as the component being a release. If REPO_NAME was a release, it would make sense to use that as the variable name instead of having a comment to explain the variable name.

I'm not questioning whether or not the component should be checked for updates, I'm questioning the usefulness of the comment. Maybe if it explained why it is being added that would be a useful comment but what is there simply rewords the code.